test/Login.test.ts

```diff
@@ -86,10 +86,9 @@ it('Can Log in',  async () => {
 
     //create a key
     const assetKeys = new Keys();
-    //const assetWallet = new Wallet(assetKeys.privateKey);
-    
-    //const pubKey = utils.computePublicKey(assetWallet.publicKey, true);
     console.log("Asset pubkey", assetKeys.publicKey);
+    console.log("Before update DidDoc: ", await iam.getDidDocument())
```

Comment:
```suggestion
```
Best not to have the `console.log` in the test. IMO, it clutters the test output.